src/sidebar/store/index.js

```diff
@@ -31,21 +31,15 @@
  *  3. Checking that the UI correctly presents a given state.
```

Comment:
There's a comment above here that says

> *  4. Other parts of the app can subscribe to changes in the app state.
> *     This is used to to update the UI etc.

Does this still apply?

Also does this comment block belong here-seems like it should get moved to the tests instead?
> * Tests for a given action consist of:

Reply:
Yes. The API of the `store` service has not changed from the point of view of the rest of the application, so I left this initial block unchanged.

This PR is purely about making it easier to write store modules and tests.